src/components/eip4626/user/VaultWithdraw.tsx

```diff
@@ -1,50 +1,50 @@
-import { useFormik } from "formik";
-import { EvmAddress, VaultInfoProps } from "@/types/types";
 import {
+  Text,
+  Flex,
+  Input,
+  Button,
   Alert,
   AlertDescription,
   AlertIcon,
-  AlertTitle,
-  Button,
-  FormControl,
-  FormHelperText,
-  FormLabel,
-  Heading,
-  HStack,
-  NumberInput,
-  NumberInputField,
-  VStack,
 } from "@chakra-ui/react";
-import { useReadHederaVaultShare } from "@/hooks/eip4626/useReadHederaVaultShare";
+import Icon from "@/components/Icon";
+import { ActionName, TransactionResult } from "@/components/TransactionResult";
+import { EvmAddress } from "@/types/types";
+import { useWriteHederaVaultApprove } from "@/hooks/eip4626/mutations/useWriteHederaVaultApprove";
 import { useWriteHederaVaultWithdraw } from "@/hooks/eip4626/mutations/useWriteHederaVaultWithdraw";
-import { useReadBalanceOf } from "@/hooks/useReadBalanceOf";
 import { formatBalance, formatNumberToBigint } from "@/services/util/helpers";
-import { useWriteHederaVaultApprove } from "@/hooks/eip4626/mutations/useWriteHederaVaultApprove";
+import BigNumber from "bignumber.js";
+import { useFormik } from "formik";
+import { useState, useMemo } from "react";
 import { useQueryClient } from "@tanstack/react-query";
-import { useReadHederaVaultUserContribution } from "@/hooks/eip4626/useReadHederaVaultUserContribution";
+import { useReadHederaVaultShare } from "@/hooks/eip4626/useReadHederaVaultShare";
+import { useReadBalanceOf } from "@/hooks/useReadBalanceOf";
+
+type VaultWithdrawProps = {
+  vaultAddress: EvmAddress;
+};
 
-export function VaultWithdraw({ vaultAddress }: VaultInfoProps) {
+export const VaultWithdraw = ({ vaultAddress }: VaultWithdrawProps) => {
```

Comment:
Small thing, but I'd align with the rest of the project and export function instead of const for consistency